test/docker/Dockerfile

```diff
@@ -13,16 +13,16 @@ FROM docker:latest as docker
 ############################################
 # Image to create go binaries
 FROM golang:1.12.5 as golang
-ENV     GO111MODULE=on
-ENV     GOPROXY=https://proxy.golang.org
```

Comment:
Why did you remove the proxy ? I think it was supposed to speed things up and make sure deleted repos don't affect us.

Reply:
`build istio.io/test-infra/toolbox/ci2gubernator: cannot load github.com/google/go-github/github: unexpected status (https://proxy.golang.org/github.com/google/go-github/@v/v0.0.0-20171108000855-8c08f4fba5e0.zip): 400 Bad Request` if it is included. Not sure why

Reply:
Did it fail once or consistently ? We should check what is the speed impact, and maybe look for a different proxy.

Reply:
It is failing every time. I tried 8 days ago, 4 days ago and today. Not sure why though.. either way this should only effect the docker image being built, so speed is not a major concern right?

Reply:
I just tried pulling it locally this morning using the `GOPROXY=https://goproxy.io` and that worked fine. I have had issues even getting the _istio/istio_ modules to download with _https://proxy.golang.org_ (the later is still a Beta I believe).